### DISCUSSION TITLE
Determination of OCP wrt stoichiometric ratio (x)

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2105] [Q&A] [answered]
Hello all, 
I am having the data of cathode as OCP v/s Capacity (mAh/g) obtained from experimental procedure.
I have to input this data into pybamm as OCP vs stoichiometric ratio (x). 
I have the data as follows:
a) Maximum capacity (mAh/g),
b) Non-extractable charge (mAh/g),
c) Reversible capacity (mAh/g) = (Maximum capacity - Non-extractable charge),
d)  Maximum & Min stoichiometric index,

I tried dividing the capacity with maximum capacity values to get the index values from 0 to 1 as stoichiometric ratio, (I am not sure if this the proper procedure to do in pybamm).
Kindly let me know if anyone has tried this before in pybamm.

Regards,
Manjunath N

**Comment by brosaplanella** [comment, marked as answer]
Hi Manjunath,

You need indeed to convert the capacity into stoichiometry. To do so, you need to figure out (probably from literature) how you can map two voltage points to a given stoichiometry, so you can find out the linear transformation to use. We did that in the [Chen 2020](https://iopscience.iop.org/article/10.1149/1945-7111/ab9050/meta) paper so you can find more details there.
